### DISCUSSION TITLE
We are deprecating discussions

### DISCUSSION THREAD

**Discussion by AucaCoyan** [espanso/espanso#1934] [General]
Hello! In the last meeting of espanso discord we agreed to drop discussions. They are about the same of having issues, and we don't really like to look in a lot of places to see what is going on. We already have github issues, discord, reddit and discussions aren't bringing anything new.
Of course we will not delete anything! Just preventing to create new ones and reworking the issues to welcome to all topics.
See you around!

**Comment by reitzig** [comment]
How should we interpret closing & locking without comment? "Won't do", "open issue if you still care", or something else?

**Comment by AucaCoyan** [comment]
It depends on each case. The mayority of cases are talking about an old version of espanso (2022 or older). And in some cases in which its proponed a new feature, it's a good use case, but we don't have the throughput to make it possible 😢 . Especially in short term (a year from now)
I've rather have fewer issues and try to fix them, than lots of issues and prioritize between a lot of cases.

Also, I would like to announce somewhere that espanso is excellent as it is now. I'm not planning adding many more features, but better integrations and bugfixes everywhere. This is to stabilize the software and have a consistent product (not a sunset of the project).
You are free to open an issue if you want!
